src/dockerflow/logging.py

```diff
@@ -65,6 +65,7 @@ class JsonLogFormatter(logging.Formatter):
             "processName",
             "relativeCreated",
             "stack_info",
+            "taskName",
```

Comment:
Is this a drive-by, or specifically related to 3.12 somehow?

Reply:
This is specific, python 3.12 adds this fields and we don't want it in the mozlog fields